File: snippets/fsharp/System/UInt64/ToString/tostring1.fs
module tostring1

// <Snippet1>
let value = 163249057uL
// Display value using default ToString method.
printfn $"{value.ToString()}\n"      

// Define an array of format specifiers.
let formats = [| "G"; "C"; "D"; "F"; "N"; "X" |]
// Display value using the standard format specifiers.
for format in formats do
    printfn $"{format} format specifier: {value.ToString format,16}" 
// The example displays the following output:
//       163249057
//       
//       G format specifier:        163249057
//       C format specifier:  $163,249,057.00
//       D format specifier:        163249057
//       F format specifier:     163249057.00
//       N format specifier:   163,249,057.00
//       X format specifier:          9BAFBA1
// </Snippet1>

File: snippets/csharp/System/UInt32/ToString/tostring1.cs
// <Snippet1>
using System;

public class UInt32ToStringExample1
{
    public static void Run()
    {
        uint value = 1632490;
        // Display value using default ToString method.
        Console.WriteLine(value.ToString());
        Console.WriteLine();

        // Define an array of format specifiers.
        string[] formats = { "G", "C", "D", "F", "N", "X" };
        // Display value using the standard format specifiers.
        foreach (string format in formats)
            Console.WriteLine($"{format} format specifier: {value.ToString(format),16}");
    }
}
// The example displays the following output:
//       1632490
//
//       G format specifier:          1632490
//       C format specifier:    $1,632,490.00
//       D format specifier:          1632490
//       F format specifier:       1632490.00
//       N format specifier:     1,632,490.00
//       X format specifier:           18E8EA
// </Snippet1>
